### ISSUE TITLE
phnoptr is updated but yet positive phenotype error is being showed.

### ISSUE THREAD

**Issue by db7007** [akoyabio/phenoptr#20]
Respected Sir,
Greetings of the season.
I have the following error after merging the data and then I tried to consolidate it. 

> remotes::install_github('akoyabio/phenoptr@9000')
Skipping install of 'phenoptr' from a github remote, the SHA1 (7d5dceaa) has not changed since last install.
  Use `force = TRUE` to force installation
> remotes::install_github('akoyabio/phenoptrReports@9000')
Skipping install of 'phenoptrReports' from a github remote, the SHA1 (af94a14c) has not changed since last install.
  Use `force = TRUE` to force installation

> phenoptrReports:::addin_20_consolidate()

Listening on http://127.0.0.1:4908
Reading "CD4+ PHENOTYPING ALL_Merge_cell_seg_data". 
Data is already in microns, no conversion performed                                                                 
Adding Slide ID column to Merge_cell_seg_data.txt
Writing report for "CD4+ PHENOTYPING ALL_Merge_cell_seg_data". 
Loading required package: UpSetR
Reading "CD8+ PHENOTYPING ALL_Merge_cell_seg_data". 
Data is already in microns, no conversion performed                                                                 
Adding Slide ID column to Merge_cell_seg_data.txt
Writing report for "CD8+ PHENOTYPING ALL_Merge_cell_seg_data". 
Reading "foxp3+lymphoid_Merge_cell_seg_data". 
Data is already in microns, no conversion performed                                                                 
Adding Slide ID column to Merge_cell_seg_data.txt
Writing report for "foxp3+lymphoid_Merge_cell_seg_data". 
Reading "NKP46+PHENOTYPING ALL_Merge_cell_seg_data". 
Data is already in microns, no conversion performed                                                                 
Adding Slide ID column to Merge_cell_seg_data.txt
Warning: Error in split_phenotype_column: No positive phenotypes found in column "Phenotype".
  3: shiny::runApp
  2: shiny::runGadget
  1: phenoptrReports:::addin_20_consolidate

Ran on:
RStudio 2021.09.1+372 "Ghost Orchid" Release (8b9ced188245155642d024aa3630363df611088a, 2021-11-08) for macOS
Mozilla/5.0 (Macintosh; Intel Mac OS X 12_0_1) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.10 Chrome/69.0.3497.128 Safari/537.36

**Comment by AB-Kent** [comment]
What are the values in the Phenotype column of "NKP46+PHENOTYPING ALL_Merge_cell_seg_data" ? Does it contain any values ending in "+" ?

**Comment by shwetajohari** [comment]
Good afternoon Sir, 
Facing a similar issue, there are no values that end with + in the column. 
How do I change that please? Do I have to re-phenotype on inform? 

Looking forward to your response. 

Yours sincerely,
Shweta Johari

**Comment by AB-Kent** [comment]
Yes, to use phenoptr, your positive phenotypes must have names ending in +. You can re-name them in inForm or edit the inForm output files change this.

**Comment by shwetajohari** [comment]
Respected Sir,
I have added the “+” sign as well as there is no space or hypen between the
name and sign.
In my other project also I can only see one positive Cd8+ but in the images
there are so many.
How do I go ahead with This?
Looking forward to your reply.
Yours sincerely,
Shweta Johari

**Comment by AB-Kent** [comment]
Please ask your local FAS for help.
